tools/upgraders.js

```diff
@@ -239,6 +239,35 @@ the guide about breaking changes here:`,
     packagesFile.writeIfModified();
   },
 
+  "1.4.3-split-accounts-packages": function (projectContext) {
+    const packagesFile = projectContext.projectConstraintsFile;
+    const usingAccountsUI = !! packagesFile.getConstraint("accounts-ui");
```

Comment:
In #8346 (an alternative to this PR), I didn't make the addition of `<service>-config-ui` dependent on whether `accounts-ui` was present.

I did this to ensure that if someone wasn't using `accounts-ui` but was using some `templating` aspect of `<service>` (included via `accounts-<service>`), that their experience would remain the same.  Admittedly, very cautious and I couldn't decide what the reach/feasibility of this use-case was, so err-ed on the side of "staying exactly as is" (i.e. if you had `templating` before, you kept `templating`, unless you explicitly removed the newly added `<service>-config-ui` packages).

That said, doing it the way you have here might have the added benefit of automatically getting rid of the Blaze dependency for those that don't need it (as indicated by their lack of `accounts-ui`) – I'm okay with this.